integration/taskcluster-hook.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Nit: You could add a test like the bugbug test_hooks.py test to make sure the hook definition is valid.

Reply:
Done